docs/howto/local-checkouts.md

```diff
@@ -39,10 +40,72 @@ storage, resulting in cost savings.
 
 <iframe width="420" height="315" src="https://www.youtube.com/embed/afgQnmesLZM"></iframe>
 
-## **lakectl local**: The way to work with lakeFS data locally  
+## **lakeFS Mount**: Efficiently expose lakeFS Data as a local directory
+
+⚠️ lakeFS Mount is currently in preview. There is no installation required, please [contact us](https://info.lakefs.io/thanks-lakefs-mounts) to get access.
+{: .note }
+
+#### Prerequisites:
+
+- A working lakeFS Server running either [lakeFS Enterprise](../enterprise) or [lakeFS Cloud](../cloud)
+- You’ve installed the [`lakectl`](../reference/cli.html) command line utility: this is the official lakeFS command line interface, on top of which lakeFS Mount is built.
+- lakectl is configured properly to access your lakeFS server as detailed in the configuration instructions
+
+### Mounting a lakeFS reference as a local directory
```

Comment:
It will be good to mention that lakeFS Mount has a Git integration

Reply:
will do! probably in a separate  PR